src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -76,68 +108,105 @@ const BioAndDMSetting: React.FC<BioAndDMSettingProps> = ({
-        <ToggleSwitch
-          value={isAnonymity}
-          onValueChange={toggleSwitchAnon}
-          labelLeft="Allow anonymous messages?"
-          circleActiveColor={COLORS.signed_primary}
-          activeTextColor={COLORS.signed_primary}
+        <Divider
+          style={{
+            marginVertical: 12
+          }}
         />
-      </TouchableOpacity>
-
-      {isAnonymity && (
-        <TouchableOpacity
-          onPress={toggleSwitchAnonAllowFollowing}
-          style={styles.toggleSwitchAnonFollowing}>
-          <ToggleSwitch
-            value={isAllowFollowingSendDM}
-            onValueChange={toggleSwitchAnonAllowFollowing}
-            labelLeft="Only allow anon DMs from users you follow?"
-            circleActiveColor={COLORS.signed_primary}
-            activeTextColor={COLORS.signed_primary}
-          />
+        <Text style={styles.bioText}>{addDotAndRemoveNewline(bio)}</Text>
+      </View>
+      <View
+        style={{
+          backgroundColor: COLORS.white,
+          borderRadius: 12,
+          padding: 12,
+          marginHorizontal: 12,
+          shadowColor: 'rgba(0, 0, 0, 0.04)',
+          shadowOffset: {
+            width: 0,
+            height: 1
+          },
+          shadowOpacity: 1,
+          shadowRadius: 2,
+          elevation: 2,
+          marginTop: -50,
+          borderWidth: 1,
+          borderColor: COLORS.gray
+        }}>
+        <Text
+          style={{
+            fontSize: 12,
+            fontWeight: '400',
+            color: COLORS.gray500,
+            marginBottom: 12
+          }}>
+          Other users will be able to reply to your prompt and direct message you.
+        </Text>
+        <TouchableOpacity onPress={toggleSwitchAnon}>
+          <CheckBoxCustom value={isAnonymity} label="Allow anonymous messages?" />
         </TouchableOpacity>
-      )}
+
+        {isAnonymity && (
+          <>
+            <Divider style={{marginVertical: 12, backgroundColor: COLORS.gray}} />
+            <TouchableOpacity onPress={toggleSwitchAnonAllowFollowing}>
+              <CheckBoxCustom
+                value={isAllowFollowingSendDM}
+                label="Only allow anon DMs from users you follow?"
+              />
+            </TouchableOpacity>
+          </>
+        )}
+      </View>
     </View>
   );
 };
 
 const styles = StyleSheet.create({
+  promptTitleContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center'
+  },
+  promptTitle: {
+    fontSize: 14,
+    fontWeight: '600',
+    color: COLORS.white
+  },
   container: {
-    backgroundColor: COLORS.signed_primary,
+    backgroundColor: COLORS.default_signed_secondary,
     borderRadius: 15,
     paddingHorizontal: 12,
-    marginTop: 20
+    paddingVertical: 12,
+    marginTop: 10,
+    paddingBottom: 50
   },
   editPromptLabel: {color: COLORS.signed_secondary, textDecorationLine: 'underline'},
   bioText: {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [191-220]

The changes to the styles object seem fine. However, the marginBottom style property on line 219 could be considered a magic number. As mentioned before, consider defining these values as constants. 

Overall, the changes made to `BioAndDMSetting.tsx` seem to improve the user interface and functionality of the profile screen. Just remember to avoid magic numbers and consider moving reusable components to a shared directory.


<!-- This is an auto-generated comment by OSS CodeRabbit -->